autoplex/auto/phonons/jobs.py

```diff
@@ -327,7 +327,9 @@ def dft_phonopy_gen_data(
     jobs = []
     dft_phonons_output = {}
     dft_phonons_dir_output = []
-    supercell_matrix = reduce_supercell_size(structure, **supercell_settings)
+    supercell_matrix = supercell_settings.get("supercell_matrix")
```

Comment:
either add the supercell_matrix to each of the jobs and then figure out which structure in the list the supercell matrix belongs to or just count the structures. Same for the random_structure_generation

Reply:
sounds good. I guess I was thinking this in the wrong direction.